### PR TITLE
UI fix

### PR DESCRIPTION
**Pull request by kazob1998**
Fixing following bugs:
- sample name shows undetermined even when code lab is set
- d18O_cel is not captivated when a csv file is imported
- Refer to [bug report](https://docs.google.com/spreadsheets/d/1q6yD0WvVlHzWFe5U8Gb1uSXC1T3JjKj369T5RrosbpY/edit?resourcekey=0-OyvbQda0Fosbus5iACgClg&gid=1511627527#gid=1511627527 ) for more details. 
- test change: https://river-sky-386919--pr98-ui-fix-dpb30v4r.web.app/login

### CODE REVIEW

File: sample_tracking/app/edit/page.tsx [hunk 1]
```diff
@@ -117,6 +117,7 @@ export default function Edit() {
         const sampleData = {
             ...updatedFormData,
             visibility: 'private',
+            d18O_cel: updatedFormData.d18O_cel ? updatedFormData.d18O_cel.map((value: string) => parseFloat(value)) : [],
```
Comment (benwulfe) [inline sample_tracking/app/edit/page.tsx:120]: did this ever work without d18O_cel?  Was this an oversight or did we t-test on a different column? (O_wood, but that does not all sound right to me)

File: sample_tracking/app/import-samples.tsx [hunk 2]
```diff
@@ -203,11 +210,15 @@ export default function ImportSamples() {
...
                 let foundErrors = false;
+                let errorsArray:any = [];
+                let row = 0
                 results.data.forEach((result) => {
                     const errors = validateImportedEntry(result as Sample, errorMessages);
+                    console.log("Errors found while validating imported entry" + errors)
                     if (errors.length > 0) {
                         result.errors = errors;
                         foundErrors = true;
+                        errorsArray.push({row: row, error: errors})
```
Comment (benwulfe) [inline sample_tracking/app/import-samples.tsx:221]: there was also a feature that Josh did where we would re-export the improperly formatted file with the errors added as a new column.  Is that still working?

**Review by benwulfe — COMMENTED** [review]
Looks good, just a few questions

File: sample_tracking/app/edit/page.tsx (hunk 1 above)
Comment (kazob1998) [reply sample_tracking/app/edit/page.tsx:120]: I believe this was an oversight from our end.

File: sample_tracking/app/import-samples.tsx (hunk 2 above)
Comment (kazob1998) [reply sample_tracking/app/import-samples.tsx:221]: yes it is still working. The error bar will provide a link to download the csv with the errors.

**Review by benwulfe — APPROVED** [review]

**Pull request merged**